test/framework/src/main/java/org/elasticsearch/test/junit/listeners/ReproduceInfoPrinter.java

```diff
@@ -67,7 +67,7 @@ public void testFailure(Failure failure) throws Exception {
         boolean isBwcTest = Boolean.parseBoolean(System.getProperty("tests.bwc", "false"));
 
         // append Gradle test runner test filter string
-        b.append("'" + task + "'");
+        b.append("\"" + task + "\"");
```

Comment:
Single quotes doesn't work in Windows. Double quotes should work everywhere

Reply:
I think we went single quotes here so we don't accidentally do glob pattern matching or something. At least, that's why I'd have used single quotes if it were me.

Reply:
Looking into this, I reached the conclusion that this "shouldn't" affect gradle, as this is the CLI escaping mechanism? Not sure, and not an expert. But as it works in Windows, Mac, and Linux (from your other comment)... This is just a error message anyway :dancing-blob: